starter_scripts/Dockerfile

```diff
@@ -27,11 +25,14 @@ COPY exasol_integration_test_docker_environment /integration_test_docker_environ
 COPY pytest_itde /integration_test_docker_environment/pytest_itde
 COPY poetry.lock /integration_test_docker_environment/poetry.lock
 COPY pyproject.toml /integration_test_docker_environment/pyproject.toml
-COPY setup.py /integration_test_docker_environment/setup.py
 COPY LICENSE /integration_test_docker_environment/LICENSE
 COPY README.rst /integration_test_docker_environment/README.rst
 
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH /root/.local/bin:$PATH
+RUN poetry config virtualenvs.create false
```

Comment:
did you try if this really works, with a much older version of poetry it didn't work a few years ago